cyder/core/system/models.py

```diff
@@ -34,8 +34,7 @@ def filter_by_ctnr(ctnr, objects=None):
         return objects.filter(ctnr=ctnr)
 
     def check_in_ctnr(self, ctnr):
```

Comment:
`get_ctnrs` is inhereited from `BaseModel`. `check_in_ctnr` wasn't added in this pull request, I believe it was added to fix a `_has_perm()` issue from a long time ago. Since we're moving away from using the active container to check permissions, it might be unnecessary now... but I haven't checked that.